x-pack/plugins/index_management/__jest__/client_integration/home/home.test.ts

```diff
@@ -54,15 +54,22 @@ describe('<IndexManagementHome />', () => {
     });
 
     describe('tabs', () => {
-      test('should have 2 tabs', () => {
+      test('should have 4 tabs', () => {
         const { find } = testBed;
-        const templatesTab = find('templatesTab');
-        const indicesTab = find('indicesTab');
-
-        expect(indicesTab.length).toBe(1);
-        expect(indicesTab.text()).toEqual('Indices');
-        expect(templatesTab.length).toBe(1);
-        expect(templatesTab.text()).toEqual('Index Templates');
+        const allTabs = [
+          { label: 'Index Templates', element: find('templatesTab') },
+          { label: 'Indices', element: find('indicesTab') },
+          { label: 'Data Streams', element: find('data_streamsTab') },
+          { label: 'Component Templates', element: find('component_templatesTab') },
+        ];
+
+        expect(allTabs.length).toBe(4);
+
+        allTabs.forEach((tab) => {
+          expect(tab.element).toBeTruthy();
+          expect(tab.element.length).toBe(1);
+          expect(tab.element.text()).toEqual(tab.label);
```

Comment:
I think we only need this assertion, since it will fail if either of the assertions on lines 69 or 70 fail.